packages/ra-core/src/auth/useLogoutIfAccessDenied.spec.tsx

```diff
@@ -40,7 +39,7 @@ const authProvider: AuthProvider = {
 };
 
 const TestComponent = ({ error }: { error?: any }) => {
-    const [loggedOut, setLoggedOut] = useSafeSetState(false);
+    const [loggedOut, setLoggedOut] = React.useState(false);
```

Comment:
```suggestion
    const [loggedOut, setLoggedOut] = useState(false);
```

Could you please import it